extensions/wikia/OasisLight/scripts/top.js

```diff
@@ -0,0 +1,29 @@
+Wikia = {};
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "let", "const" or "var" keyword to this declaration of "Wikia" to make it explicit. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS2703)